index.js

```diff
@@ -117,22 +165,44 @@ var Umzug = module.exports = redefine.Class({
 
         return all.filter(function (migration) {
           return executedFiles.indexOf(migration.file) === -1;
-        }).sort(function (a, b) {
```

Comment:
Fair point.I wonder if I actually added that due to a bug report. Maybe windows problems?